src/device-state.ts

```diff
@@ -344,6 +350,16 @@ export async function setTarget(target: TargetState, localSource?: boolean) {
 			await config.set({ targetStateSet: true }, trx);
 		});
 	});
+
+	const targetStateConfig = await fsUtils.exists(TARGET_STATE_CONFIG_DUMP);
+	if (!targetStateConfig) {
+		await fsUtils.touch(TARGET_STATE_CONFIG_DUMP);
```

Comment:
What was the logic behind this? I don't think this is necessary, right?

Reply:
It isn't. Thanks!